apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx

```diff
@@ -1,15 +1,13 @@
 import React, {ChangeEvent, useState} from 'react';
-import {useDispatch, useSelector} from 'react-redux';
+import {useDispatch} from 'react-redux';
```

Comment:
it doesn't seem to make a difference, but we also have a typed version of `useDispatch`, `useAppDispatch` in `reduxHooks`